pkg/getty/README.md

```diff
@@ -10,7 +10,7 @@ If you want a console getty, add the following to your `moby.yml`:
 ```
 services:
   - name: getty
-    image: "linuxkit/getty:e36b5e794256c2a56c000035f73a4d46f7ffb3e6"
```

Comment:
we don't want to use the `sha256` because the build itself is not reproducible in all cases (binaries containing timestamps and such things) so we take the hash from all the input and take care that the build does not pull un-deterministic stuff from the network (either by using the alpine base image or pulling very specific version from the network).

We used to explicitly hash the contents of the input files, but that process was error prone (occasionally missed a file) and complex (tar pipelines) and settled for the git tree hash (which is the hash of the contents of the directory) as the next best thing.

Having the README with example hashes in the directory then  becomes a slight issue as we tend to update hash with `git grep ... | sed ...` and thus alter the content hash of the directory creating the circle...

In general, I would prefer having documentation in `./docs` anyway as it's easier to have it one place